### ISSUE TITLE
Is there a requirement to use date=type for dates. 

### ISSUE THREAD

**Issue by CalvinRodo** [wet-boew/wet-boew#8073]
Because there are no plans to override the edge's current native date-picker (as per #7594) is there a requirement that dates must be type=date. 

Our employees use keyboard input for entering dates and requiring them to use the up and down arrows to enter a date is not acceptable, and with our department moving to edge as our default web-browser in the near future we need a solution to this problem. 
 
As well is it possible to either recommend an accessible date-picker or allow developers to use wet's existing polyfill on type=text inputs. 

@rlambert27

**Comment by kodecount** [comment]
Personally I think the simplest thing you can do is just use a text input, but create a custom jQuery Validation (and have a server side validation (for when jQuery not available)) to make sure the date enter is a valid date. It not hard making custom jQuery validation rules for form validation.

**Comment by LaurentGoderre** [comment]
Then you loose mobile support though....

**Comment by kodecount** [comment]
True, it could be difficult with mobile (kinda forget about it sometimes).

**Comment by CalvinRodo** [comment]
In this case Mobile Support is not an issue as I'm asking in regards to internal facing sites only where the users are only going to be able to access the site on our approved default browser (IE Edge). So this is a pretty specific scenario.
